src/_providers.tf

```diff
@@ -0,0 +1,20 @@
+terraform {
+  required_version = ">= 1.0"
+  required_providers {
+    massdriver = {
```

Comment:
Specify the AWS provider as well so you can set the version. Like this: https://github.com/massdriver-cloud/aws-guardduty-detector/blob/main/src/_providers.tf#L8-L11